### PR TITLE
MiniAB Pinda Probe Mount

### PR DESCRIPTION
**Pull request by DrDespa1r**
* [X ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X ] Submitted STL files are oriented and scaled properly for printing.
  * [X ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
A replacement MiniAB cowling that adds a Pinda probe mount. 
#### How was it tested? 
I've been running it for about 6 months on my v0 now, and I think there are a few other uses using it now as well that haven't complained to me about it. 
#### Any background context you want to provide?
Extra probe option for the miniAB for people who don't want to use the stock endstop or some kind of klicky probe. 
#### Screenshots (if appropriate)
Contained in the folder, hope that's ok!

### CODE REVIEW

**Comment by cbc02009** [comment]
Looks good to me.

**Review by cbc02009 — APPROVED** [review]

File: printer_mods/MrDespair/MiniAB_Pinda/readme.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Mini Afterburner Pinda Probe Mount
+==============================
+
+A modified dragon toolhead cowling to mount a pinda (or superpinda, or vinda) probe to your VO.1! (Mosquito will come someday if I get this working).
```
Comment (KiloQubit) [inline printer_mods/MrDespair/MiniAB_Pinda/readme.md:4]: `VO.1` should use the digit `0` instead of the letter `O`

**Review by KiloQubit — CHANGES_REQUESTED** [review]

**Comment by DrDespa1r** [comment]
Changes should be done now!

**Review by KiloQubit — APPROVED** [review]

**Comment by KiloQubit** [comment]
Looks like one of your STLs is corrupted, please fix:

```
Validating STL printer_mods/MrDespair/MiniAB_Pinda/STLs/Cowling_Dragon_v3.stl
  !! Detected "Edges fixed" value of 4
  Error occurred while validating STLs.
```

**Comment by DrDespa1r** [comment]
Repaired the .stl file, sorry about that!

**Comment by hdtechk** [comment]
would like to see "Revo Voron  Cowling" added to this mod.

**Pull request merged**